angular/src/app/screens/login/login.component.ts

```diff
@@ -30,7 +32,11 @@ export class LoginComponent {
     this.userService.login(value).subscribe(
       (response: HttpResponse<Object>) => {
         if (response.status === StatusRequest.Ok) {
-          console.log(response.headers.get('Access-Token'));
+          const token = response.headers.get('Access-Token');
+          if (token) {
+            this.userService.createSession(token);
```

Comment:
You need to save access-token, uid, client. Storage it as an unique string in a single slot on localStorage 👌🏻